python/ray/air/checkpoint.py

```diff
@@ -351,6 +351,16 @@ def from_directory(cls, path: str) -> "Checkpoint":
         """
         return cls(local_path=path)
 
+    @classmethod
+    def copy_from(cls, other: "Checkpoint") -> "Checkpoint":
+        """Create a checkpoint from a generic :py:class:`Checkpoint`."""
```

Comment:
```suggestion
        """Create a checkpoint from a generic :py:class:`Checkpoint`.
        
        This can be used to create a framework-specific checkpoint (e.g., TorchCheckpoint)
        from a generic Checkpoint class, e.g., `TorchCheckpoint.from_checkpoint(chkpt)`.
        """
```